test/gurka/test_gtfs.cc

```diff
@@ -0,0 +1,137 @@
+#include "gurka.h"
+#include "just_gtfs/just_gtfs.h"
+
+#include "string.h"
+#include <gtest/gtest.h>
+
+using namespace gtfs;
+using namespace valhalla;
+
+// test to read gtfs data from sample
+TEST(GtfsExample, ReadGtfs) {
+  Feed feed("../test/data/gtfs_sample");
+  EXPECT_EQ(feed.read_agencies().code, ResultCode::OK);
+  const auto& agencies = feed.get_agencies();
+  EXPECT_EQ(agencies.size(), 1);
+  EXPECT_EQ(agencies[0].agency_id, "DTA");
+}
+
+// test to write gtfs files
+TEST(GtfsExample, WriteGtfs) {
+
+  Feed feed;
+
+  // write agency.txt
+  struct Agency ttc {
+    .agency_id = "TTC", .agency_name = "Toronto Commission", .agency_url = "http://www.ttc.ca",
+    .agency_timezone = "America/Toronto"
+  };
```

Comment:
for now this test is fine I think. eventually we might want to encapsulate the gtfs writer in a function which we can call from single test suites where we have a bit control on e.g. coordinates for certain test maps and could actually integrate them with standard gurka maps even. anyways, future stuff. so far we're only testing `just_gtfs` actually, no valhalla api yet. but also there isn't any new valhalla api yet.

ideally we'd also make sure we produce valid gtfs data. you could run this tool on the `gtfs_test` dataset: https://github.com/MobilityData/gtfs-validator. seems to be well maintained. how we'll integrate that tool in the future to make sure all our (synthetic) test gtfs data is eventually valid, not sure yet. if we take this current dataset for the time being, let's validate it manually first before moving on.